applications/CableNetApplication/custom_elements/sliding_cable_element_3D.cpp

```diff
@@ -92,9 +92,15 @@ void SlidingCableElement3D::GetDofList(DofsVectorType &rElementalDofList,
   }
 }
 
-void SlidingCableElement3D::Initialize() {
-  KRATOS_TRY
-  KRATOS_CATCH("")
+void SlidingCableElement3D::Initialize()
+{
+    KRATOS_TRY
+    if (GetProperties()[CONSTITUTIVE_LAW] != nullptr) {
+        mpConstitutiveLaw = GetProperties()[CONSTITUTIVE_LAW]->Clone();
```

Comment:
don't save it extra, use `Check` to check if it exists and the retrieve it from the Properties when you need it

Reply:
If you want to have it historic on each gp you should clone it. I don't know if that is the case...

Reply:
you are right, my bad, never mind then
Just move the check to `Check`